ai/pocs/nn_layers/model_launcher.py

```diff
@@ -0,0 +1,97 @@
+import numpy as np
+import tensorflow as tf
+from tensorflow import keras
+import json
+import sys
+
+def match_to_data(match):
+    """format the json data to a more usable format
+
+    Args:
+        match (match): A match from the json data
+
+    Returns:
+        (
+            list(dict({
+            "name": str,
+            "stat": Stat
+            })),
+            list(dict({
+            "name": str,
+            "stat": Stat
+            })),
+            int,
+            int)
+        ): The formatted match
+    """
+    first_x = []
+    second_x = []
+
+    for unit in match['first_player']['units']:
+        if unit is not None:
+            first_x.append(list(unit['stat'].values()))
+    for unit in match['second_player']['units']:
+        if unit is not None:
+            second_x.append(list(unit['stat'].values()))
+
+    sample = (
+        first_x,
+        second_x,
+        match['first_player']['score'],
+        match['second_player']['score']
+    )
+    return sample
+
+
+
+def format_value(value):
+    """build the model's trainning data from the dataset
+
+    Args:
+        value (
+            list(dict({
+                "name": str,
+                "stat": Stat
+                })),
+                list(dict({
+                "name": str,
+                "stat": Stat
+                })),
+                int,
+                int)
+        ): The dataset formatted
+
+    Returns:
+        (np.array(np.array(int)), np.array(int)): A tuple with 1) The units to train on and 2) The scores to train on
+    """
+    scores = []
+    # This file is an old poc, so the army length is hardcoded. I decided to keep those files are they are, since they are not used anymore
```

Comment:
If it's unused and old make it clear in the name of the file in a specific folder